metadata-ingestion/src/datahub/ingestion/source/sql/sql_generic_profiler.py

```diff
@@ -70,7 +70,6 @@ def generate_profile_workunits(
         self,
         requests: List[TableProfilerRequest],
         max_workers: int,
```

Comment:
we can force these to be kwargs for clarity

```suggestion
        *, max_workers: int,
```